### ISSUE TITLE
optistack warning after Acado run

### ISSUE THREAD

**Issue by timueh** [casadi/optistack#6]
Hi there,

I have existing and working Acado code in Matlab  that solves a simple NMPC problem (two-state CSTR, single input, positive definite quadratic cost). I wanted to morph this to optistack+Matlab. Borrowing much from your optistack NMPC example I got it to work. But there is some funny behavior when I call my code as follows:

1. Use optistack to solve the problem --> works fine.
2. Use Acado to solve the problem --> works fine.
3. Use optistack to solve the problem again --> I get a warning that reads

```
Warning: A value of class "uint64" was indexed with no subscripts specified. Currently the
result of this operation is the indexed value itself, but in a future release, it will be an
error. 
> In SwigRef/subsref (line 22)
  In casadi.DM (line 1113)
  In casadi.Function/call (line 790)
  In SwigRef/subsref (line 29)
  In optisolve/readoutputs (line 282)
  In optisolve/resolve (line 276)
  In optisolve (line 198)
  In P2Ex2_Cas (line 51) 
```
I use "clear all" at the beginning of every run 1.-3.

In short: everything works, but I get an optistack warning when Acado was called before. The usual MS-trick does the job -- exit Matlab. Then calling optistack again results in no warning.

It is no major issue. I thought you'd be interested anyway. Of course, I can e-mail the m-files to you.

cheers, till.
